openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateElectricEquipmentITEAirCooled.cpp

```diff
@@ -0,0 +1,265 @@
+          }
+        }
+      }
+    }
+
+  } else {
+    LOG(Warn, modelObject.briefDescription() << " not assigned to a valid space.");
+  } // end of pre-process
+
+
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::Name, modelObject.name().get());
+
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::AirFlowCalculationMethod, definition.airFlowCalculationMethod());
+  
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::DesignPowerInputCalculationMethod, definition.designPowerInputCalculationMethod());
+
+  // To match other equipment objects, the Number of Units field was dropped. Users can just use multiplier for both Watts/Area and Watts/Unit definitions
+  // this is to get rid of the warning message for leaving the Number of Units field blank.
+  idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::NumberofUnits, 1.0);
+  
+  double multiplier = modelObject.multiplier();
+
+  OptionalDouble d = definition.wattsperUnit();
+  if (d){
+    idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::WattsperUnit, (*d)*multiplier);
+  }
+
+  d = definition.wattsperZoneFloorArea();
+  if (d){
+    idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::WattsperZoneFloorArea, (*d)*multiplier);
+  }
+
+  boost::optional<Schedule> designPowerInputSchedule = modelObject.designPowerInputSchedule();
+  if (designPowerInputSchedule){
+    idfObject.setString(ElectricEquipment_ITE_AirCooledFields::DesignPowerInputScheduleName, designPowerInputSchedule->name().get());
+  }
+  
+  boost::optional<Schedule> cPULoadingSchedule = modelObject.cPULoadingSchedule();
+  if (cPULoadingSchedule){
+    idfObject.setString(ElectricEquipment_ITE_AirCooledFields::CPULoadingScheduleName, cPULoadingSchedule->name().get());
+  }
+
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::CPUPowerInputFunctionofLoadingandAirTemperatureCurveName, definition.cPUPowerInputFunctionofLoadingandAirTemperatureCurve().name().get());
+
+  if (!definition.isDesignFanPowerInputFractionDefaulted()) {
+    idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::DesignFanPowerInputFraction, definition.designFanPowerInputFraction());
+  }
+
+  d = definition.designFanAirFlowRateperPowerInput();
+  if (d) {
+    idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::DesignFanAirFlowRateperPowerInput, *d);
+  }
+
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::AirFlowFunctionofLoadingandAirTemperatureCurveName, definition.airFlowFunctionofLoadingandAirTemperatureCurve().name().get());
+
+  idfObject.setString(ElectricEquipment_ITE_AirCooledFields::FanPowerInputFunctionofFlowCurveName, definition.fanPowerInputFunctionofFlowCurve().name().get());
+
+  if (!definition.isDesignEnteringAirTemperatureDefaulted()) {
+    idfObject.setDouble(ElectricEquipment_ITE_AirCooledFields::DesignEnteringAirTemperature, definition.designEnteringAirTemperature());
```

Comment:
Our new standard is to write the OS value to IDF even if it is defaulted.  This covers the case where OS and E+ have different defaults.